app/src/main/java/gov/hhs/cdc/trustedintermediary/wrappers/TiLogger.java

```diff
@@ -0,0 +1,61 @@
+package gov.hhs.cdc.trustedintermediary.wrappers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Humble object interface for logging This class was created in order to take away the 3rd party
+ * dependency of the logger. The idea is to have the logger dependency only in this class. If there
+ * ever is a reason to use a different logger, then we only need to make the changes here.
+ */
+public class TiLogger implements MyLogger {
```

Comment:
I would do slightly different names. I would name this class something Kike `Slf4jLogger`. Basically something that indicates it is specific to SLF4J. I would rename the interface to be something like `TiLogger` or just `Logger`. Basically the interface should have the generic name that transcends any implementation.

Reply:
Yes, I agree. This was the part that I had a bit of trouble with. I didn't go with the Logger at first because of the different options for import. But I like that you pointed it out. I do like the Logger and Slf4jLogger names better.